lib/dependencies.sh

```diff
@@ -109,6 +109,11 @@ yarn_2_install() {
 
   echo "Running 'yarn install' with yarn.lock"
   cd "$build_dir" || return
+
+  if ! has_yarn_cache "$build_dir"; then
+    echo "Yarn cache is required. See https://devcenter.heroku.com/articles/migrating-to-yarn-2#make-changes-to-source-code for additional information"
```

Comment:
What do you think about changing this to:
```bash

if ! has_yarn_cache "$build_dir"; then
  monitor "yarn-2-install-with-node-modules" yarn install 2>&1
else
 ...
```

Reply:
that's what i thought of doing originally but i didn't want to be presumption with asking your team first haha.